tests/backup/backup_test.go

```diff
@@ -1240,6 +1228,15 @@ var _ = Describe("{ShareBackupWithUsersAndGroups}", func() {
 			err := Inst().S.Destroy(contexts[i], opts)
 			dash.VerifySafely(err, nil, fmt.Sprintf("Verify destroying app %s, Err: %v", taskName, err))
 		}
+
+		log.Infof("Deleting registered clusters for non-admin context")
+		for _, ctxNonAdmin := range userContexts {
+			err := DeleteCluster(SourceClusterName, orgID, ctxNonAdmin)
+			dash.VerifySafely(err, nil, fmt.Sprintf("Deleting cluster %s", SourceClusterName))
+			err = DeleteCluster(destinationClusterName, orgID, ctxNonAdmin)
+			dash.VerifySafely(err, nil, fmt.Sprintf("Deleting cluster %s", destinationClusterName))
+		}
+
```

Comment:
Here we can use DeleteCloudAccounts(make(map[string]string), "", "", ctxNonAdmin) instead of the above 4 lines

Reply:
Done